test/fixtures/internal-modules/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('internal/freelist');
```

Comment:
doesn't change a lot. we only need to test this once with any module. it could be anything other than `freelist`